composer.json

```diff
@@ -34,6 +34,10 @@
     "conflict": {
         "fzaninotto/faker": "*"
     },
+    "suggest": {
+        "ext-dom": "(\\Faker\\Provider\\HtmlLorem) Required for generating random HTML.",
+        "symfony/polyfill-iconv": "(\\Faker\\Provider\\ru_RU\\Text::realText()) Required for generating real Russian text."
```

Comment:
No point in suggesting a the polyfil. Either require the polyfill, or suggest the extension. I'd vote for the 2nd option, given the English faker stuff doesn't need those functions.

Reply:
Thanks, I changed this now to suggest ext-intl.